aspnet/security/cors.rst

```diff
@@ -37,16 +36,17 @@ Setting up CORS
 
 To setup CORS for your application you use the ``Microsoft.AspNetCore.Cors`` package. In your project.json file, add the following:
 
-.. literalinclude:: cors/sample/src/CorsExample1/project.json
-  :language: none
-  :lines: 5,6,9
-  :emphasize-lines: 2
+.. literalinclude:: cors/sample/src/CorsExamples/CorsExample1/project.json
+  :language: javascript
+  :lines: 2-12
+  :emphasize-lines: 10
```

Comment:
This is an advanced sample, should we think about just telling them what line to add **AND** not even have the version number?